unfoldingWord/translationCore
# Projects With Merge Conflicts Cannot Be Exported

#### This pull request addresses:
- Blocking merge conflict projects from being exported.
- Fixing a bug with merging a project which difference was merely a verse being deleted.
- Updating the usfm-js version.
- Fixing a bug which led to falsely identify duplicate projects by book/language code combination

#### How to test this pull request:
Run ```npm update usfm-js``` to get new usfm-js module
Try opening a project that has the same language code / book combination and ensure you get an alert to that re-importing the same project is not allowed.
Try exporting a project with merge conflicts (USFM/tS) and ensure you get an alert that you must resolve them first 
- There are merge conflict projects in the tests folder ```__tests__/fixtures/project/mergeConflicts/```


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3056)
